### DISCUSSION TITLE
PRINT_START phase error. Expected 'preheat' but found 'extruder'

### DISCUSSION THREAD

**Discussion by IdahoCreator** [jschuh/klipper-macros#234] [Q&A]
After switching board and adjusting scripts to work on new BTT SKR Pico board that I used to repalce a Voxelab Aquilla X2 with H32 chip, I started to get the following erro and cant seem to resolve. "Error evaluating 'gcode_macro _print_start_phase_check:gcode': gcode.CommandError: PRINT_START phase error. Expected 'preheat' but found 'extruder'"  I have included all of my config files for you to look at unless you know for sure what is causing this issue. It was working then I replaced board. I did set everything up for probing but found that for what ever reason the npn offset probe wasnt going to work so I went back to z endstop. I have scoured google and a lot of klipper forums and fixxed the smaller problems but this one isnt coming up.
[config-20240312-174711.zip](https://github.com/jschuh/klipper-macros/files/14579969/config-20240312-174711.zip)

**Comment by Monschtrum** [comment]
Same Problem for me after Update Klipper & Macros
